salonB.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="static/css/main.css" />
+    <script type="module" src="src/JS/main.js"></script>
+    <title>Salong A</title>
+  </head>
+  <body>
+    <header class="sec">
+      <div class="header-content-wrap">
+        <img src="static/images/logo.png" alt="logo" />
+        <nav class="header-nav">
+          <button class="header-nav-toggle"></button>
+          <menu class="header-menu">
+            <li>
+              <a class="header-menu-item active" href="/index.html">Hem</a>
```

Comment:
Class "active" on index.html should not be active if you are on salonB.html

Reply:
Fixed.